uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_install.sql

```diff
@@ -31,7 +31,7 @@ CREATE TABLE `civicrm_hrleaveandabsences_absence_type` (
      `is_sick` tinyint   DEFAULT 0 COMMENT 'A flag which is used to determine if this Absence Type can be used for a Sickness Request',
      `calculation_unit` varchar(512) NOT NULL   COMMENT 'One of the values of the Absence type calculation units option group',
      `hide_label` tinyint   DEFAULT 0 COMMENT 'This controls the visibility of the Leave Type label in the calendar and feeds.',
-      `category` varchar(10) NOT NULL COMMENT 'This is used for grouping leave types',
+     `category` int unsigned NOT NULL   COMMENT 'This is used for grouping leave types.',
```

Comment:
The SQL statement still reads NOT NULL whereas the xml file says the field is not required.

Reply:
Kindly check again. The [required](https://github.com/compucorp/civihr/pull/2870/commits/9bd60e35945379187af0532fb5ef76336180507a#diff-8ff8f4f4dbde96aab412eb3d1754ce62L186) reads true.